src/pudl/package_data/eia923/column_maps/fgd_operation_maintenance.csv

```diff
@@ -0,0 +1,19 @@
+year_index,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021,2022
+report_year,year,year,year,year,year,year,year,year,year,year,year,year,year,year,year
+plant_id_eia,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id,plant_id
+fgd_unit_id,fgd_id,fgd_id,fgd_id,fgd_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id,fgd_unit_id
```

Comment:
Possibly should be `so2_control_id_eia` to match with `emissions_control`?

Reply:
If this is the same value as the one in the emissions control table, then they should have the same column name.

Reply:
Maybe you were looking at an older version? This was changed in commit `e171327` because I had the same realization!

Reply:
I noticed you'd changed it, I was just voicing support!